OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -271,6 +271,12 @@ public interface IMove
 		bool CanEnterTargetNow(Actor self, Target target);
 	}
 
+	public interface ITemporaryBlocker
+	{
+		bool CanRemoveBlocker(Actor self, Actor blocking);
+		bool Blocks();
```

Comment:
`bool Blocks(Actor self, CPos cell)` please.  Certain actor types may want to block only certain cells.